rancher/configuration/settings/index.md

```diff
@@ -18,11 +18,97 @@ The setup determines the base URL your hosts will use to connect to the Rancher
+
+* The first file `config.yml` contains the details of your entry.
+
+     ```yaml
+     name: # Name of the Catalog Entry 
+     description: |
+        # Description of the Catalog Entry
+     version: # Version of the Catalog to be used 
+     category: # Category to be used for searching catalog entries
+     ```
+<br>
+* The second file is a the icon image for the catalog entry. The file must be prefixed with `catalogIcon-`. 
+
+For every catalog entry, there will be a minimum of three items: `config.yml`, `catalogIcon-entry.svg`, and the `0` folder, which holds the first version of the catalog entry. 
+
+#### Rancher Catalog Templates 
+
+The `docker-compose.yml` and `rancher-compose.yml` are located within the version folder number (i.e. `0`, `1`, etc.). Rancher uses these files to run [rancher-compose]({{site.baseurl}}/rancher/rancher-compose/) to launch the services into Rancher.
+
+The `docker-compose.yml` should be a file that could also be launched using `docker-compose up`. The services follow the docker-compose format.
+
+The `rancher-compose.yml` will contain additional information to help customize your catalog entries. In the `.catalog` section, there are some fields that will be required in order to have your catalog entry interprested correctly.
+
+An optional `README.md` is possible to be created, which provides a lengthy description or notes on how to use the catalog service. 
+
+
+**`rancher-compose.yml`**
+
+```yaml
+.catalog
+  name: # Name of the versioned template of the Catalog Entry 
+  version: # Version of the versioned template of the Catalog Entry 
+  description: # Description of the versioned template of the Catalog Entry
+  uuid: # Unique identifier to be used for upgrades. Please see note. 
+  questions: #Used to request user input for configuration options
+```
+<br>
+> **Note:** The `uuid` is a required parameter that is used for upgrade scenarios. Each `uuid` must be unique and each later version should be incremental. The recommended format is to use the name of the catalog entry and suffixed with a `-0`. As versions are introduced, you would increment the `-0` to `-1` for the next entry. 
+
+**Questions in the `rancher-compose.yml`**
+
+The `questions` section of `.catalog` is used to be able to show questions to the user and allow users to provide different configuration options. 
+
+Each question is a list item in the `questions` section.
+
+```yaml
+.catalog
+  questions:
+    - variable: # A single word that is used to pair the question and answer.
+      label: # The "question" to be answered.
+      description: # The description of the question to show the user how to answer the question.
+      default: # A default value that will be pre-populated into the UI
+      required: # Whether or not an answer is required. By default, it's considered `false`.
+      type: # The format type of the response. Eligible formats are `"string"`, `"int"`, `enum`,  `"multiline"`, `"service"`
```

Comment:
Maybe explain these a little bit in their own section? Might not be clear to everyone.